.circleci/config.yml

```diff
@@ -30,9 +31,20 @@ jobs:
           name: Setup application
           command: cd src/api; bundle exec rake dev:bootstrap RAILS_ENV=test
       - run: mkdir ~/rspec
+      - run:
+          name: Download code climate test reporter
+          command: cd src/api; curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter; chmod +x ./cc-test-reporter
```

Comment:
Not sure we should do this on every build, sounds like we should package this and include it in the container.

Reply:
Yeah, this is just for initial testing. Packaging is probably not possible because we only have the binaries provided by codeclimate AFAIK. But we could include the binary into the container with Kiwi.